test/extensions/common/wasm/wasm_runtime_factory_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "envoy/registry/registry.h"
+
+#include "source/extensions/common/wasm/wasm_runtime_factory.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Wasm {
+
+class FakeRuntimeFactory : public WasmRuntimeFactory {
+public:
+  WasmVmPtr createWasmVm() override { return nullptr; }
+  std::string name() const override { return "envoy.wasm.runtime.mock"; }
+};
+
+class WasmRuntimeFactoryTest : public testing::Test {};
+
+TEST_F(WasmRuntimeFactoryTest, DestructFactory) {
```

Comment:
I think this test should check the name and the WasmVmPtr, also deleting a factory is not common usage.

Reply:
The purpose of this test case is just for testing the default "destructor" of the factory, not the name nor WasmVmPtr. Such functions (`name` or `createWasmVm`) seems to be covered by the other test cases when I look at the coverage data.

What do you think?

Reply:
This test also feels a little odd to me. The destructor in question is a default destructor, it feels like overkill to me to have to test it explicitly. I'd lean towards removing this test case altogether unless we really need the coverage stats.